content/changelog-stable/index.html

```diff
@@ -57,6 +57,20 @@
   <div id="rc" style="display:none;">
   </div>
 
+<h3><a name=v2.32.1>What's new in 2.32.1</a> (2016-12-21)</h3>
```

Comment:
This also needs a section about what's new since 2.19.4. See 2.19.1 changelog for comparison.